client/src/app/site/pages/meetings/pages/participants/pages/participant-import/components/participant-import-list/participant-import-list.component.ts

```diff
@@ -12,9 +12,11 @@ import { ParticipantImportService } from '../../services';
     styleUrls: [`./participant-import-list.component.scss`]
 })
 export class ParticipantImportListComponent extends BaseViaBackendImportListMeetingComponent {
-    public possibleFields = Object.keys(participantHeadersAndVerboseNames);
+    public possibleFields = Object.keys(participantHeadersAndVerboseNames).sort(
+        (a, b) => participantColumnsWeight[a] - participantColumnsWeight[b]
+    );
```

Comment:
see comments in `participant-csv-export.service.ts` and `participant-import/definitions/index.ts`

Reply:
Done.